### PR TITLE
SDKS-928 : Update README.md for XCode support statement

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

**Review by spetrov — DISMISSED** [review]
Looks good to me! 👍🏻

File: README.md [hunk 1]
```diff
@@ -28,7 +28,6 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
 * ForgeRock Identity Platform
     * Access Management (AM) 6.5.2+
 * iOS 10 and above   
-* Xcode 11.0 or later
```
Comment (differentthings) [inline README.md:31]: to be consistent, let's leave it as "Xcode", without a version number

**Review by differentthings — COMMENTED** [review]
I left one comment. Please let me know if you disagree.

**Review by spetrov — APPROVED** [review]
Looks good!

**Pull request merged**